ui/src/components/Home/Home.jsx

```diff
@@ -1,16 +1,19 @@
 import * as React from "react"
-import MessagesView from "../MessagesView/MessagesView";
+import Map from "../Map/Map"
 import LoggedOutView from "../LoggedOutView/LoggedOutView";
 import * as Bootstrap from "react-bootstrap"
 import 'bootstrap/dist/css/bootstrap.min.css';
 import "./Home.css"
 
 export default function Home({ isLoggedIn, handleLogout, handleLogin }) {
+<<<<<<< HEAD
     // If the user is logged in, go to the home page, otherwise show the login screen
+=======
+>>>>>>> ce97652 (Map Layout)
```

Comment:
This looks like the merge messages are still here? You might need to remove

Reply:
These aren't showing up on my end, let me try re-committing the code